.gitignore

```diff
@@ -5,3 +5,5 @@
 *.user
 /RetroAchievementsKey
 test_app/
+src/*_REMOTE_*.cpp
+src/clipboard.txt
```

Comment:
i realize these are to be ignored, but where do they come from such that they need to be ignored?

Reply:
clipboard.txt is file I'm using to copy and paste stuff (not trying to sound smart-alecky). <code>*_REMOTE_*.cpp</code> came from KDiff3, a merging tool I use to resolve conflicts.